### DISCUSSION TITLE
Wait for Custom Resource Readiness

### DISCUSSION THREAD

**Discussion by somasundaram1996** [helmfile/helmfile#850] [Questions]
Hi,
Is there a way that helmfile wait for the custom resource to get ready. If not I am having a script that runs on postsync to check the readiness of the custom resource. Is there a way to fail the release if custom resource is not ready either directly or through postsync hooks?

**Comment by yxxhero** [comment]
@somasundaram1996 maybe this is a feature request.

**Reply by somasundaram1996** [reply]
So there is not Workaround to fail the release if the postsync hook fails? Is there any roadmap when this feature can be included?

**Reply by somasundaram1996** [reply]
Hi @yxxhero , Can I know about the roadmap?

**Comment by jouve** [comment]
`wait` is implemented by helm , I think the feature request should be there
